packages/webapp/__tests__/AccountSecurityPage.tsx

```diff
@@ -262,10 +262,10 @@ it('should allow setting new password but require to verify session', async () =
   const submitResetPassword = await screen.findByText('Set password');
   fireEvent.click(submitResetPassword);
   await waitForNock();
-  mockSettingsValidation(params);
+  const mock = mockSettingsValidation(params);
   await verifySession();
-  const input = await screen.findByPlaceholderText('Password');
-  expect(input).toHaveValue('#123xAbc');
+  await waitForNock();
+  expect(mock.isDone()).toBeTruthy();
```

Comment:
To prove that the expectation works, I tried to comment out the line `await verifySession` - and it failed. We need the verification first so the mock will be triggered.